sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -158,6 +159,23 @@ def generate_dead_letter_entity_name(
     return entity_name
 
 
+def copy_messages_if_needed(messages):
```

Comment:
Nit: let's either make a note here that it creates a sendable message in docstring, or adjust the helper name to specify that.  I only had terrible ideas so feel free to override my suggestion.

```suggestion
def copy_messages_to_sendable_if_needed(messages):
```

Reply:
docstring added and name changed, thanks for the suggestion